server/src/graphql/user/resolvers.ts

```diff
@@ -1,12 +1,26 @@
-const queries={};
-
+import { prismaClient } from "../../lib/db";
+const queries={
+  getUser : async(_:any,payload:{email:string,profileImageUrl:string})=>{
+      const User=await prismaClient.user.findMany({where:{
+        profileImageUrl: null
+      }})
+      // console.log(User);
+      return User
+  }
+};
 const mutations={
-    createUser:(_:any,
+    createUser: async(_:any,
```

Comment:
this server will not be getting any password, it is completely handled in nextjs